src/main/scala/net/cakesolutions/CakeStandardsPlugin.scala

```diff
@@ -46,6 +46,8 @@ object CakeStandardsPlugin extends AutoPlugin {
             "-Xlint",
             "-Yno-adapted-args",
             "-Ywarn-dead-code",
+            "-Ywarn-value-discard",
```

Comment:
Generally, we've found it a good thing to have around - e.g. because we've had futures and such like thrown away on projects. We've tended to handle this by then adding in code that explicitly allows the value to be discarded, but forces the user to state the values type.

For testing, I can see arguments for both keeping this setting (e.g. tests also have futures etc.) and removing it on the test scope (e.g. because you often need to throw values away in tests). Looking at code from a recent project, we did find ourselves having to deal with value discards, but not to the degree whereby it was patently painful. However, maybe this all comes down to the type of testing code used on a project?

Perhaps we should see how others think on this matter?